code/src/Style/Container.css

```diff
@@ -0,0 +1,32 @@
+.container {
```

Comment:
I like that you have divided your CSS in different files, it makes it easy for me to follow.